### PR TITLE
AuthForm and AddTransactionForm improvements

### PR DESCRIPTION
**Pull request by oozywaters**

### CODE REVIEW

**Comment by oozywaters** [comment]
Ready for review.

File: react/e2e/private/notificationsMenu/notificationsMenu.test.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('Notifications menu', () => {
     const page2 = await browser.newPage()
     await login(page2, TEST_ACCOUNTS[1], process.env.JEST_SECRET)
 
-    await page2.waitForSelector(SELECTORS.notificationsCounter)
+    await page2.waitFor(3000)
```
Comment (mxfactorial) [inline react/e2e/private/notificationsMenu/notificationsMenu.test.js]: can wait until [current value != beforeCounter](https://stackoverflow.com/questions/50373158/puppeteer-wait-for-element-to-have-certain-value)

**Comment by mxfactorial** [comment]
coverage dropped from https://github.com/systemaccounting/mxfactorial/runs/600613043?check_suite_focus=true

coverage-drop

for example, screens/HomeScreen/decorators.js missing coverage

**Comment by oozywaters** [comment]
Ready for review.

**Pull request merged**